### ISSUE TITLE
Support parsing requirements.txt files that aren't specifically named "requirements.txt"

### ISSUE THREAD

**Issue by alex** [google/osv-scanner#370]
It's somewhat common in the Python ecosystem to have a requirements.txt file with a different name, e.g. `ci-requirements.txt` or `requirements/docs.txt`.

Having a way to parse these would be super valuable. I imagine either the filename support could be expanded, or an additional CLI arg to specify the lockfile format could work.

**Comment by G-Rath** [comment]
This is [already supported](https://google.github.io/osv-scanner/usage/#specify-lockfiles):

```
osv-scanner --lockfile 'requirements.txt:ci-requirements.txt' --lockfile 'requirements.txt:requirements/docs.txt'

**Comment by alex** [comment]
Well, that's embarrassing that I didn't realize this. Thanks.

On Sat, May 6, 2023 at 1:27 PM Gareth Jones ***@***.***> wrote:
>
> Closed #370 as completed.
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.Message ID: ***@***.***>



-- 
All that is necessary for evil to succeed is for good people to do nothing.
